### PR TITLE
Tweak to Carrion Explosive Spider

### PR DESCRIPTION
**Pull request by Harrivchaw**
## About The Pull Request
i ded pls nerf
Carrion spiders now have a delay in explosion, two second, more importantly they now pop out when activated, glowing a bright orange hue along with a warning message in the chat.

## Why It's Good For The Game
i ded pls nerf
Make it so carrion explosive spiders are able to be countered, instead of being haha i click you dead

## Changelog
:cl:
tweak: Carrion explosive spider now has a delay and pops out of the victim's body when it's activated
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm
Comment (TheShown911) [inline code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:13]: I would just move the current implant to the turf instead of making a new one.

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm
Comment (TheShown911) [inline code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:31]: Don't think `O` is used here.

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm
Comment (TheShown911) [inline code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:20]: If people are fast enough they will be able to squash the spider before it explodes so you could do something like `biteszadusto?.prime()` to avoid a runtime.

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm
Comment (TheShown911) [inline code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:9]: You could change this to time defines like `2 SECONDS`.

**Review by TheShown911 — COMMENTED** [review]
I approve the idea.

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm
Comment (Harrivchaw) [reply code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:20]: changed

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm
Comment (Harrivchaw) [reply code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:9]: made it that way so it resembles grenades timer, will change

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm
Comment (Harrivchaw) [reply code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:31]: copied grenade proc, will check

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm
Comment (Harrivchaw) [reply code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:13]: some pointers would be nice, not all that familiar with all the basic proc and function yet

Comment (TheShown911) [reply code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:13]: Eris itself doesn't have any documentation, so sadly the only way to learn is trial and error. It's normal do not do everything perfectly the first try. I don't know the proc to detatch implants from the top of my head, but cruciform or surgery code should contain it.

**Review by genessee596 — APPROVED** [review]
YEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEES

still needs code approval from maintainer

File: code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm
Comment (TorinoFermic) [inline code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm:9]: Leave one space between 2 and SECONDS

**Review by TorinoFermic — APPROVED** [review]

**Pull request merged**
